lib/Applify.pm

```diff
@@ -149,6 +154,19 @@ OPTION:
 
   print "Usage:\n";
 
+  if ($self->_has_subcommands){
+    my $subcmds = [ sort { $a->{name} cmp $b->{name} } values %{$self->{subcommands}} ];
+    my $width = 0;
+    for my $subcmd(@$subcmds) {
+      my $length = length $subcmd->{name};
+      $width = $length if $width < $length;
+    }
+    print "\n    ", File::Basename::basename($0), " [command] [options]\n";
+    print "\ncommands:\n";
+    printf("    %-${width}s  %s\n", @{$_}{'name', 'desc'}) for @$subcmds;
```

Comment:
This  code that use `$subcmds` looks a bit clunky, but I can't come up with anything better myself.

Reply:
trimmed the for loop - works with `@options` too.

```perl
($width) = sort { $b <=> $a } map { length $_->{name} } @options;
```